src/components/ComposedModal/_composed-modal.scss

```
@@ -0,0 +1,59 @@
+@import '../../globals/vars';
+@import '../../globals/layout';
+
+.#{$iot-prefix}--composed-modal--modal {
+  .bx--modal-container {
```

Need to use the carbon prefix
```suggestion
  .#{$prefix}--modal-container {
```